### DISCUSSION TITLE
Supabase library for Arduino

### DISCUSSION THREAD

**Discussion by gigabytejayesh** [supabase/supabase#1189] [Feature Requests]
Supabase, Finally we have an alternative to Firebase! Awesome

I have been working on some Internet of Things project where I used Firebase real time database.

But alternative to that,
**Creating a Supabase library for Arduino will be great idea** to use database for Internet of Things.

**Comment by kiwicopple** [comment]
Agreed, it would be cool! Perhaps you can add this as an idea in the Client Libraries thread? https://github.com/supabase/supabase/discussions/5

**Comment by deandreamatias** [comment]
Had this library unofficial https://github.com/jhagas/ESP32-Supabase
